parser.go

```diff
@@ -10,18 +10,47 @@ import (
 	"io/ioutil"
 )
 
-func ParseFile(path string) (*GoFile, error) {
-	source, err := ioutil.ReadFile(path)
-	if err != nil {
-		return nil, err
+// ParseFiles parses files at the same time
+func ParseFiles(paths []string) ([]*GoFile, error) {
+	var files []*ast.File
+	var fsets []*token.FileSet
+	for _, p := range paths {
```

Comment:
couldn't this function be:
```go
func ParseFiles(paths []string) ([]*GoFile, error) {
    goFiles := make([]*GoFile, len(paths))
    for i, path := range paths {
        goFile, err := ParseSingleFile(p)
        if err != nil {
            return nil, err
        }

        goFiles[i] = goFile
    }
}
```

Reply:
Updated it. In golang, we can define structs in different files in the same package. We need to pass more `files []*ast.File` in `parseFile` to check.